package.json

```diff
@@ -8,10 +8,16 @@
     "lint": "yarn run lint:js && yarn run lint:scss",
     "lint:js": "standardx 'app/assets/javascripts/**/*.js' 'spec/javascripts/**/*.js'",
     "lint:scss": "stylelint app/assets/stylesheets/",
-    "jasmine:prepare": "RAILS_ENV=test bundle exec rails assets:clobber assets:precompile",
-    "jasmine:ci": "yarn run jasmine:prepare && yarn run jasmine-browser-runner runSpecs",
+    "rails:precompile-assets": "RAILS_ENV=test bundle exec rails assets:clobber assets:precompile",
+    "nyc:instrument": "yarn run nyc instrument public/assets/frontend .nyc_output/assets",
+    "jasmine:prepare": "yarn run rails:precompile-assets && yarn run nyc:instrument",
+    "nyc:report": "yarn run nyc report",
```

Comment:
Small nitpick: should we group similar scripts together? or put the scripts in alphabetical order?

Non-blocking as it feels more like something we should have a general consensus on and tooling in place to manage this, ES Lint for example - https://eslint.org/docs/latest/contribute/package-json-conventions#order

Reply:
Yeah, we can change the ordering after it's decided in another tiny PR.